sycl/plugins/unified_runtime/ur/adapters/cuda/command_buffer.cpp

```diff
@@ -106,6 +106,90 @@ static void setCopyParams(const void *SrcPtr, const CUmemorytype_enum SrcType,
   Params.Depth = 1;
 }
 
+// Helper function for enqueuing memory fills
+static ur_result_t enqueueCommandBufferFillHelper(
+    ur_exp_command_buffer_handle_t CommandBuffer, void *DstDevice,
+    const CUmemorytype_enum DstType, const void *Pattern, size_t PatternSize,
+    size_t Size, uint32_t NumSyncPointsInWaitList,
+    const ur_exp_command_buffer_sync_point_t *SyncPointWaitList,
+    ur_exp_command_buffer_sync_point_t *SyncPoint) {
+  ur_result_t Result;
+  std::vector<CUgraphNode> DepsList;
+  UR_CALL(getNodesFromSyncPoints(CommandBuffer, NumSyncPointsInWaitList,
+                                 SyncPointWaitList, DepsList));
+
+  try {
+    auto N = Size / PatternSize;
```

Comment:
```suggestion
    size_t N = Size / PatternSize;
```
According to LLVM coding style guide, I would avoid using `auto` here because the resulting type is not obvious. There might be other occurrences in your patch ...

Reply:
I replaced "auto" by explicit types when resulting types were ambiguous.